### ISSUE TITLE
rootfs.cpio not created

### ISSUE THREAD

**Issue by tinito** [fhunleth/buildroot-edison#2]
This is a minor issue: for some reason rootfs.cpio isn't created when building the image following README instructions. The build process stops with this error:

```
buildroot-edison/output/build/linux-3.10.17/scripts/gen_initramfs_list.sh: Cannot open '/home/tino/test/edison/buildroot-edison/output/images/rootfs.cpio'
make[2]: *** [usr/initramfs_data.cpio] Error 1
```

creating the file by hand with `touch output/images/rootfs.cpio` and calling make again allows the build process to end.

I think that the problem is in buildroot-edison/linux/linux.mk, where the command `touch $(BINARIES_DIR)/rootfs.cpio` should be outside the `$(if $(BR2_TARGET_ROOTFS_INITRAMFS)` block, but I'm new to buildroot so I'm not sure about it.

**Comment by fhunleth** [comment]
I haven't looked at this in months. The changes I made to Buildroot can't be accepted upstream and I was unable to convince Intel to make changes on their end. I remember Edison support being very frustrating, but I don't remember the issue that you're seeing. Did you run `make edison_defconfig` first? Also, the rootfs.cpio is supposed to contain something, so just touching it will break things down the line.

**Comment by tinito** [comment]
Yes I did run make edison_defconfig first. I just touched rootfs.cpio as this is what the buildroot-edison/linux/linux.mk makefile does, and the rootfs actually works if I boot with the kernel built by yocto. But I'm still not able to flash the buildroot kernel and rootfs (see my other issue)

**Comment by fhunleth** [comment]
Then I don't know. I'm really sorry to say this, but given the total lack of any support from Intel, I'm not interested in putting any time into the Edison any more. If things change, I'll certain update these PRs so that you get notified.

**Comment by tinito** [comment]
I absolutely understand your point, Intel (un)support is really frustrating.
Thanks for your effort, I'll fork your repo and try to solve the issues digging into yocto sources as you did.

**Comment by andy-shev** [comment]
@tinito
 
JFYI, that Edison works now with upstream U-Boot and almost upstream kernel (if you need some stuff like USB Dual Role detection, couple of patches would be needed). So, BuildRoot instructions in my repo: https://github.com/andy-shev/buildroot/tree/intel/board/intel/common
